src/checkout/checkout-store-status-selector.ts

```diff
@@ -413,7 +424,7 @@ export function createCheckoutStoreStatusSelectorFactory(): CheckoutStoreStatusS
             isUpdatingBillingAddress: state.billingAddress.isUpdating,
             isUpdatingSubscriptions: state.subscriptions.isUpdating,
             isContinuingAsGuest: state.billingAddress.isContinuingAsGuest,
-            isUpdatingShippingAddress: state.shippingStrategies.isUpdatingAddress,
+            isUpdatingShippingAddress: isUpdatingShippingAddress(state),
```

Comment:
Please keep the existing `isUpdatingShippingAddress` as it is and create a new method i.e. `isShippingStepPending` instead. You can use `isCustomerStepPending` and `isPaymentStepPending` as a reference.